addon/utils/legislation-types.js

```diff
@@ -18,8 +18,6 @@ const LEGISLATION_TYPES = {
   wetboek: 'https://data.vlaanderen.be/id/concept/AardWetgeving/Wetboek',
   'gecoordineerde wetten':
     'https://data.vlaanderen.be/id/concept/AardWetgeving/GecoordineerdeWetten',
-  'gecoördineerde wetten':
```

Comment:
you can't just remove this, we still want to match it

Reply:
It's the one that was duplicated